nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/operations/document/AddPermission.java

```diff
@@ -55,9 +62,9 @@
 
     public static final String ID = "Document.AddPermission";
 
-    public static final String NOTIFY_KEY = "notify";
+    protected static final String NOTIFY_KEY = "notify";
```

Comment:
I'm not sure we want to move from `protected` to `public`? This breaks the compatibility.

Reply:
Thanks for the warning!